validate/setup-guides/aws/createEC2.md

```diff
@@ -33,26 +33,38 @@ Please like and subscribe to support NetMet's work and to be alerted to new cont
 
 ---
 
-This is a longer drawn out process, but we can make it through it. Let's begin creating our instance!
+## EC2 CREATION
 
-#### SERVICES 
+### Wizards
+The video utilized the classic EC2 wizard setup process.  The `step-by-step` in the below documentation will show both the classic wizard and the new setup wizard.  Please use the right side navigation links to skip to the desired guide.
+
+This is a longer process, but we can make it through it. Let's begin creating our instance!
+
+### Open Compute EC2 Console 
 Click on the`Services` button to start our process.
 
-![](/img/validator_nodes/node-aws-ec2-1.png)
+![](/img/validator_nodes/node-aws-ec2-services1.png)
+
+From the new drop down box, choose the option **`Compute`**.
 
-From the new drop down box, choose the option **`EC2`** from the **compute service** section.
+![](/img/validator_nodes/node-aws-ec2-services2.png)
 
-![](/img/validator_nodes/node-aws-ec2-2.png)
+Choose the option **`EC2`** from the **Compute** section.
 
+![](/img/validator_nodes/node-aws-ec2-services3.png)
+
+### Region
 
 We need to pick a `region` to host our EC2 instance.
 
 ![](/img/validator_nodes/node-aws-ec2-3.png)
 
+## Begin building EC2
+
 :::info IMPORTANT INFORMATION
-We will need an **`a1.xlarge`** (or better) as our `instance type`. This instance type is **not** available in all regions.
+We will need an **`t2.2xlarge`** or **`a1.2xlarge`** (or better) as our `instance type`. This instance type is **not** available in all regions.  You can move between regions to find which regions offer with instance types.
 
-( `US-West-2 Oregon` has this instance type available. )
+The US regions may not offer `a` type instances.  **We will use the `t2.2xlarge` for our tutorial**.  This offers `32Gib` of RAM verses the required `16Gib`; however, this can only create better performance on our future Node.
 :::
 
 If we don't see our Instances dashboard. Choose the `Instances` from the left side menu.
```

Comment:
I think we can remove the "Note" below. I don't think anyone would be confused by that since that's the default.